### PR TITLE
[[ Bug ]] Modile is not a platform

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

**Comment by peter-b** [comment]
@livecode-vulcan review ok a7285bf

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok a7285bfca458a87c1cec9a402b831dc29e234bf0

File: docs/dictionary/command/import-widget.lcdoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ Introduced: 8.0
 
 OS: mac,windows,linux,ios,android
 
-Platforms: desktop,web,modile
+Platforms: desktop,web,mobile
```
Comment (montegoulding) [inline docs/dictionary/command/import-widget.lcdoc:13]: But how will I deply the app to my modile?... you very nearly deleted `web,` ;-)

**Comment by livecode-vulcan** [comment]
😎 test success a7285bfca458a87c1cec9a402b831dc29e234bf0
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2777)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2786)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2772)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2788)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2794)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2798)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2816)

**Pull request merged**